### ISSUE TITLE
Error looking up credentials

### ISSUE THREAD

**Issue by Iiridayn** [knoxfighter/arcdps-killproof.me-plugin#27]
When I launch the extension window with release v2.11.0 I get one line of:
`URL: https://killproof.me/api/kp/<account>?lang=en -- Status:0 -- Response:  -- ErrorMessage: schannel: AcquireCredentialsHandle failed: SEC_E_INTERNAL_ERROR (0x80090304) -- Reason:  -- StatusLine: ` per player in the group/squad, where I've replaced the various user accounts with `<account>` in that string. This is consistent, not intermittent.

Reverting to v2.10.0 works as normal, so this appears to be a new bug or regression. It is also possibly specific to Linux, as it does not appear to have been reported before. I am happy to test builds, but am not able to fiddle with it myself as I lack familiarity with the relevant build tools, and suspect they would not work well in Linux.

There appear to have been some whitespace changes between releases, complicating visual inspection of differences; as the developer you are probably familiar with anything key that may have changed between releases.

**Comment by Iiridayn** [comment]
The latest update causes the game to lock up instead as soon as it starts to look up records, necessitating a client kill and restart.

**Comment by Iiridayn** [comment]
The latest update still locks up the game as soon as it starts to look up records, necessitating a client kill and restart.

**Comment by Sarmion** [comment]
Yeah, my game also freezes after enabling the addon, and then I get an arcdps log crash.
